docs/content/_index.md

```diff
@@ -10,16 +10,33 @@ draft: false
 
 ## What is Guardicore Infection Monkey?
 
-The Infection Monkey is an open source security tool for testing a data center's resiliency to perimeter breaches and internal server infection. The Monkey uses various methods to self propagate across a data center and reports success to a centralized Monkey Island Command and Control server.
+The Infection Monkey is an open source breach and attack simulation tool for testing a data center's resiliency to perimeter breaches and internal server infection.
+Infection Monkey will help you test implemented security solutions and will provide visibility of the internal network through the eyes of an attacker.
+
+Infection Monkey is free and can be downloaded from [our homepage](https://infectionmonkey.com/).
 
 ![Infection Monkey Documentation Hub Logo](/images/monkey-teacher.svg?height=400px "Infection Monkey Documentation Hub Logo")
 
-The Infection Monkey is comprised of two parts:
+## How it works
+
+Architecturally Infection Monkey tool is comprised of two parts:
+
+* Monkey Agent (Monkey for short) - a safe, worm like binary program which scans, spreads and simulates attack techniques on the **local network**.
+* Monkey Island Server (Island for short) - a C&C web server which serves GUI for users and interacts with Monkey Agents.
+
+User runs Monkey Agent on the Island server machine or distributes Monkey Agent binaries on the network manually. Based on 
+the configuration parameters, Monkey Agents scan, propagate and simulate attackers behaviour on the local network. All of the 
+information gathered about the network is aggregated in the Island Server and displayed once all Monkey Agents are finished.
```

Comment:
```suggestion
The user can run the Monkey Agent on the Island server machine or distribute Monkey Agent binaries on the network manually. Based on 
the configuration parameters, Monkey Agents scan, propagate and simulate attackers behavior on the local network. All of the 
information gathered about the network is aggregated by the Island Server and displayed once all Monkey Agents are finished.
```

Reply:
Not sure what the first part solves.

Reply:
It flows a little better and is more consistent with some of the edits that @MarketingYeti has been making.

Reply:
New version is longer. Also, we're describing a workflow, and the word `can` is unnecessary, since there's no case where you deploy island but don't run monkey. Changed regardles